### ISSUE TITLE
[FIX] All Outline.com links are dead

### ISSUE THREAD

**Issue by tycrek** [tycrek/degoogle#525]
### Checklist

- [X] **I have read the guidelines in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md)**
- [ ] I am affiliated with this alternative

### Alternative name

[anything with an Outline.com link]

### Error

Outline.com seems to have shut down and is now a generic park page.

### Proposed changes

Replace all existing Outline.com links with the original article links, if available.

### Additional details

See #498

**Comment by freddy-m** [comment]
More generally, there seems to be a lot of clutter in the list. It could be worth removing some articles/links and just consolidating a few really good ones?

**Comment by tycrek** [comment]
I fully agree with that; a lot of the links (especially in News) are quite old and may not be relevant anymore.

**Comment by freddy-m** [comment]
Had a look at the [news articles](https://github.com/tycrek/degoogle#news-articles) section, basically everything can be removed aside from maybe '[You Are Now Remotely Controlled](https://www.nytimes.com/2020/01/24/opinion/sunday/surveillance-capitalism.html)' and '[Google will always do evil](https://www.engadget.com/2018-05-24-google-will-always-do-evil.html)'. In addition, it could be worth adding '[Privacy matters because it empowers us all](https://aeon.co/essays/privacy-matters-because-it-empowers-us-all)' by Carissa Veliz. As for the blog posts, not sure what you want to keep - but I'd love it if you added my post on Google: [Try not to be evil](https://jottings.lol/2022/12/try-not-to-be-evil)
